src/moveit_opw_kinematics_plugin.cpp

```diff
@@ -248,6 +248,37 @@ struct LimitObeyingSol
   }
 };
 
+void MoveItOPWKinematicsPlugin::expandIKSolutions(std::vector<std::vector<double> >& solutions) const
+{
+    const std::vector<const robot_model::JointModel*>& ajms = joint_model_group_->getActiveJointModels();
+    for (size_t i = 0; i < ajms.size(); ++i ) {
+        const robot_model::JointModel* jm = ajms[i];
+        if (jm->getVariableBounds().size() > 0) {
+            for (auto& bounds : jm->getVariableBounds()) {
+                // todo: what to do about continuous joints
```

Comment:
Just curious, do you know any examples of robots with an "opw" structure with continuous joints?

Reply:
I think there might be some but I cannot remember/find any. This would only be about j6 in any case I think ... very vague :wink:

Anyway, this code would just not create additional solutions for this code but does not run into any endless loop

Reply:
Avoiding an endless loop is always a good idea :)